readme.md

```diff
@@ -40,6 +40,13 @@ The following dependencies need to be installed on your system:
 	* Don't use `3.7.6` it causes an error while building, see #10696.
 * Microsoft Visual Studio 2019 Community, Version 16.3 or later:
 	* Download from https://visualstudio.microsoft.com/vs/
+	*When installing Visual Studio, you can import the config:
```

Comment:
There needs to be a space after the asterisk to make them render as bullets in a list.  For each of these lines.

```suggestion
	* When installing Visual Studio, you can import the config:
```